Makefile

```diff
@@ -46,6 +46,12 @@ jjb_test:
 	jenkins-jobs --ignore-cache test jenkins/ci.suse.de:jenkins/ci.suse.de/templates/ cloud* openstack* > /dev/null
 	jenkins-jobs --ignore-cache test jenkins/ci.opensuse.org:jenkins/ci.opensuse.org/templates/ cloud* openstack* > /dev/null
 
+cisd_deploy:
+	jenkins-jobs --conf /etc/jenkins_jobs/jenkins_jobs-cisd.ini update jenkins/ci.suse.de:jenkins/ci.suse.de/templates/ cloud* openstack*
+
+cioo_deploy:
+	jenkins-jobs --conf /etc/jenkins_jobs/jenkins_jobs-cioo.ini update jenkins/ci.opensuse.org:jenkins/ci.opensuse.org/templates/ cloud* openstack*
```

Comment:
Its just a copy of the line that deploys the ci.s.d jobs. Just in case we will have jobs that use our reserved prefix. But I can remove it if you prefer.